.github/profile/README.md

```diff
@@ -0,0 +1,20 @@
+![az digital 2](https://github.com/az-digital/.github/assets/az-digital-2.png)
+
+
+# Welcome to Arizona Digital! 🌵
+A team of web-focused volunteers known as Arizona Digital meets weekly to build and test products like Arizona Bootstrap and Arizona Quickstart. Contributions are what keep Arizona Bootstrap moving and improving, and are in everyone's best interest.
```

Comment:
Is the second sentence in this paragraph supposed to reference Arizona Digital (instead of Arizona Bootstrap)?
```suggestion
A team of web-focused volunteers known as Arizona Digital meets weekly to build and test products like Arizona Bootstrap and Arizona Quickstart. Contributions are what keep Arizona Digital moving and improving, and are in everyone's best interest.
```